model/job/Job.php

```diff
@@ -0,0 +1,131 @@
+<?php
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2018 (original work) Open Assessment Technologies SA (under the project TAO-PRODUCT);
+ *
+ */
+
+namespace oat\taoScheduler\model\job;
+
+use \DateTimeInterface;
+use oat\oatbox\PhpSerializable;
+use oat\taoScheduler\model\SchedulerException;
+
+/**
+ * Class Job
+ * @package oat\taoScheduler
+ * @author Aleh Hutnikau, <hutnikau@1pt.com>
+ */
+class Job implements JobInterface
+{
+
+    /**
+     * @var string
+     */
+    private $rRule;
+
+    /**
+     * @var DateTimeInterface
+     */
+    private $startTime;
+
+    /**
+     * @var callable
+     */
+    private $callback;
+
+    /**
+     * Schedule an event
+     *
+     * @param string $rRule Recurrence rule (@see https://tools.ietf.org/html/rfc5545#section-3.3.10)
+     * @param DateTimeInterface $startTime
+     * @param $callback Callback to be executed.
+     *                  Also can be an array with tao service identifier and method name (e.g. ['taoExt/MyService', 'doSomething'])
+     */
+    public function __construct($rRule, DateTimeInterface $startTime, $callback)
```

Comment:
Maybe we need to add third key to the array, which contain custom params for the callback method? It's will be very helpful if we need add scheduled task with some params.